plugins/inputs/mqtt_consumer/README.md

```diff
@@ -18,6 +18,10 @@ and creates metrics using one of the supported [input data formats][].
     "sensors/#",
   ]
 
+  ## The message topic will be stored in a tag specified by this value.  If set
+  ## to the empty string no topic tag will be created.
```

Comment:
s/to the empty string/to an empty string,/ reads better

Reply:
There is only one empty string, generally referred to as the empty string.  Check out the usage here: https://en.wikipedia.org/wiki/Empty_string